### PR TITLE
Fix runtime + Bible healing Vamps and Cult

### PR DESCRIPTION
**Pull request by ghost**
- Will now check for targeted mob having a mind before doing the macro. I think it should be expanded to the macro, but I'm fixing a runtime here
- Adding a comment to notify the fact that Bible nextAttackDelay() just doesn't work, no idea why, you can spam click it thus
- Cult, Vamp and Thralls are no longer healed by Bible whacks

### CODE REVIEW

**Comment by gbasood** [comment]
> Adding a comment to notify the fact that Bible nextAttackDelay() just doesn't work, no idea why, you can spam click it thus
> - user.delayNextAttack(5) //Extra delay

what

**Comment by ghost** [comment]
Didn't work IIRC, Bible can be spam-clicked

There's a lot of issues with delays, another is that you can skip delays on everything with hotkeys, allowing instant full grab window slams for example

**Comment by gbasood** [comment]
Okay, sure, but we should fix the cause of afterattack not working instead of removing the delay from the non-working proc.

**Comment by ghost** [comment]
It is in my understanding, too, that the delay is inbuilt to the very base of attack procs

AKA it's supposed to cause a one second delay

And here I have no idea whatsoever of why it doesn't work

**Comment by clusterfack** [comment]
You cant skip delays on hotkeys anymore, as long as there is a delay placed somewhere now.

**Comment by gbasood** [comment]
Unless you're going to fix the attack delay not working, leave the attack delay code in, even if it doesn't do anything right now because of the fact that afterattack doesn't work. Don't rip shit out at random.

**Comment by clusterfack** [comment]
Change those returns to return 1

**Comment by gbasood** [comment]
Actually that's probably what caused afterattack to not work
I think it expects a value of 1 to trigger afterattack

File: code/game/objects/items/weapons/storage/bible.dm [hunk 1]
```diff
@@ -149,7 +156,6 @@
 
 //We're done working on mobs, let's check if we're blessing something else
 /obj/item/weapon/storage/bible/afterattack(atom/A, mob/user as mob)
-	user.delayNextAttack(5) //Extra delay
```
Comment (gbasood) [inline code/game/objects/items/weapons/storage/bible.dm:152]: Third time I've said this, why is this being removed?
If your answer is "afterattack doesn't work" then that's not a good reason, re-add it

Comment (ghost) [inline code/game/objects/items/weapons/storage/bible.dm:152]: I moved the delay on the healing to the normal attack from from afterattack

Comment (gbasood) [inline code/game/objects/items/weapons/storage/bible.dm:152]: Well now that we know why it didn't work you can put it back.

**Pull request merged**